File: GreatOutdoor/GreatOutdoor.Entities/SalesPerson.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GreatOutdoor.Entities
{
    // Sales Person Interface
    public interface ISalesPerson
    {
        string SalesPersonName { get; set; }
        string SalesPersonEmail { get; set; }
        string SalesPersonMobile { get; set; }
        int SalesPersonID { get; set; }
        string SalesPersonPassword { get; set; }

    }

    //Sales Person Class
   public class SalesPerson:ISalesPerson
    {

        public string SalesPersonName { get ; set ; }
        public string SalesPersonEmail { get; set; }
        public string SalesPersonMobile { get; set; }
        public  int SalesPersonID { get; set; }
        public string SalesPersonPassword { get; set; }

    }
}

File: GreatOutdoors/src/app/Models/salesPerson.ts
export class SalesPerson
{
  id: number;
  salesPersonID: string;
  salesPersonName: string;
  salesPersonMobile: string;
  email: string;
  password: string;
  creationDateTime: string;
  lastModifiedDateTime: string;

  constructor(ID: number, SalesPersonID: string, SalesPersonName: string, SalesPersonMobile: string, Email: string, Password: string, CreationDateTime: string, LastModifiedDateTime: string)
  {
    this.id = ID;
    this.salesPersonID = SalesPersonID;
    this.salesPersonName = SalesPersonName;
    this.salesPersonMobile = SalesPersonMobile;
    this.email = Email;
    this.password = Password;
    this.creationDateTime = CreationDateTime;
    this.lastModifiedDateTime = LastModifiedDateTime;
  }
}


